src/common/Navigation.js

```diff
@@ -0,0 +1,11 @@
+import Link from "next/link";
+
+function Navigation(props) {
```

Comment:
We could utilize an alternative function name that better explains the use of this component.

Also, I believe you're trying to create a component that renders multiple navigation links not one. That being said, props would prop, and it could be something more descriptive like "link"

Reply:
I have changed the function name to ReusableNavLink